### PR TITLE
updated neofetch to 1.6

### PR DESCRIPTION
**Pull request by iandrewt**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

### CODE REVIEW

File: Library/Formula/neofetch.rb [hunk 1]
```diff
@@ -12,11 +12,14 @@ class Neofetch < Formula
     sha256 "ca7ebd89b1a2b1fc2036088bf3bdc08205ae5152f14ba3b1a81315dd8b39efb2" => :mavericks
   end
 
+  depends_on "screenresolution" => :recommended
+  depends_on "imagemagick" => :recommended
```
Comment (dunn) [inline Library/Formula/neofetch.rb]: How important are these dependencies? My feeling is they should be `:optional`.

**Comment by iandrewt** [comment]
`screenresolution` makes the script about doubly as fast, but there is a fallback to `system_profiler` if the user decides not to install it

`imagemagick` provides one of the highlight features of Neofetch, the image instead of ascii. Although not strictly needed, we do say in the first line of our README that the script prints an image (by default the current wallpaper) next to the system info

**Comment by mikemcquaid** [comment]
@iandrewt Seems fine, thanks.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `neofetch 1.6`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md). Thanks!

**Pull request closed without merging**